bip-0388.mediawiki

```diff
@@ -224,9 +239,10 @@ Common single-signature account patterns:
 * <tt>sh(wpkh(@0/**))</tt> (nested segwit).
 * <tt>tr(@0/**)</tt> (taproot single-signature account).
 
-Common multisignature schemes:
+Common multisig schemes:
 * <tt>wsh(multi(2,@0/**,@1/**))</tt> - SegWit 2-of-2 multisignature, keys in order.
```

Comment:
One last remaining use of "multisignature" if you wanted to convert them all to "multisig" in commit be5029f3224db8d5217a92a1ee7c90f000cf3d83

```suggestion
* <tt>wsh(multi(2,@0/**,@1/**))</tt> - SegWit 2-of-2 multisig, keys in order.
```